docs/installation.md

```diff
@@ -62,7 +62,7 @@ $ eval $(docker-machine env default)
     $ docker build -f public.Dockerfile -t amundsen-frontend:latest .
 
     # in ~/<your-path-to-cloned-repo>/amundsensearchlibrary
-    $ docker build -f Dockerfile -t amundsen-search:latest .
+    $ docker build -f public.Dockerfile -t amundsen-search:latest .
```

Comment:
Actually the dockerfile in search repo is named as https://github.com/lyft/amundsensearchlibrary/blob/master/Dockerfile. Maybe we should change that Dockerfile name to public.Dockerfile to be consistent across all the repos?

cc @jinhyukchang

Reply:
Consistency FTW 👍

Reply:
@rosejcday @jornh let us know if you have the time to open the pr to help the filename in search repo :) Otherwise I could do it later in the day :)

Reply:
@feng-tao Do you want me to change the name to be consistent with the others from [Dockerfile](https://github.com/lyft/amundsensearchlibrary/blob/master/Dockerfile) to public.Dockerfile? If so, I can do this and make a pr.

Reply:
yes :) thanks @rosejcday

Reply:
I submitted a PR `Docker file name change #13` for the file to be changed due to consistency.

Reply:
just merge it, thanks @rosejcday